### PR TITLE
Update OWNERS_ALIASES file.

### PR DESCRIPTION
**Pull request by jlojosnegros**
Update OWNERS_ALIASES file to resemble team changes.

### CODE REVIEW

File: OWNERS_ALIASES [hunk 1]
```diff
@@ -23,6 +23,8 @@ aliases:
     - vitus133
     - kenyis
     - nishant-parekh
+    - marioferh
+    - jlojosnegros
```
Comment (yanirq) [inline OWNERS_ALIASES:27]: lets add also Tal-or to the list

**Comment by jlojosnegros** [comment]
Hi @yuvalk 

If you have some time to spare please take a look at this minor change and approve it if it's ok.

Thanks in advance.

**Pull request merged**